service/src/main/java/org/databiosphere/workspacedataservice/dao/PostgresJobDao.java

```diff
@@ -12,6 +12,7 @@
 import java.util.List;
 import java.util.Optional;
 import java.util.UUID;
+import javax.annotation.Nullable;
```

Comment:
Shouldn't this be the Spring annotation?

```suggestion
import org.springframework.lang.Nullable;
```

Reply:
hm, looking through our codebase we have both - and both seem to work; both delegate to the same underlying com.google.code.findbugs:jsr305 `@Nonnull(when…`. I've switched to using the springframework annotation exclusively.

Reply:
Ahh interesting!  I'd be in favor of switching over to the core java version globally if that's of interest.